app/models/piece.rb

```diff
@@ -1,7 +1,7 @@
-require 'pry'
 class Piece < ActiveRecord::Base
   belongs_to :user
   belongs_to :game
+  has_many :types
```

Comment:
@leamerluck What does Line 4 do? I'm a bit confused 'cuz it seems to point to a nonexistent types table in the database.